ui/src/app.tsx

```diff
@@ -191,6 +195,9 @@ function ChatRoutes({ state, location, isMobile, isSmall }: RoutesProps) {
       {state?.backgroundLocation ? (
         <Routes>
           <Route path="/about" element={<AboutDialog />} />
+          <Route path="/grid" element={<Grid />} />
+          <Route path="/app/:desk" element={<AppModal />} />
+          <Route path="/app/:desk/info" element={<TileInfo />} />
```

Comment:
Should probably use suspense for each of these and load dynamically since they're kinda secondary

Reply:
Pushing this now.